lib/lita/adapters/slack.rb

```diff
@@ -26,11 +26,10 @@ def mention_format(name)
       # Starts the connection.
       def run(&block)
         return if rtm_connection
-
+        Lita.logger.info("Starting to build RTM connection")
         @rtm_connection = RTMConnection.build(robot, config)
-
-        yield if block_given?
```

Comment:
Why is the yield call deleted? I suggest keep this PR just to logging changes to keep it clear and simple.

Reply:
(I understand this change now that I read through the whole PR)